xbmc/windowing/rpi/VideoSyncPi.cpp

```diff
@@ -32,7 +33,7 @@ bool CVideoSyncPi::Setup(PUPDATECLOCK func)
 {
   UpdateClock = func;
   m_abort = false;
-  g_Windowing.Register(this);
+  CServiceBroker::GetWinSystem().Register(this);
```

Comment:
CVideoSyncPi object is owned by WinSystem. Passing the parant object on creation is a better desing. No need to involde ServiceBroker.
Not mission critical for now. You may want to do this later.